tools/code-owners-parser/CodeOwnersParser/CodeOwnerEntry.cs

```diff
@@ -40,12 +38,12 @@ public bool IsValid
 
         private static string[] SplitLine(string line, char splitOn)
         {
-            return line.Split(splitOn, StringSplitOptions.RemoveEmptyEntries);
+            return line.Split(new char[] { splitOn }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Why these changes and the quotes as well?

Reply:
Because I switched to netstandard. The syntax is a bit different, I guess for backward compatibility.